package.json

```diff
@@ -14,6 +14,7 @@
     "d": "yarn-deduplicate && yarn",
     "ios": "react-native run-ios",
     "lint": "eslint --report-unused-disable-directives --cache source/ modules/ scripts/ images/",
+    "prepare": "patch -p0 -Nfsi contrib/rn-fixes.patch || true",
```

Comment:
Minor suggestion in case you want to change it: as prior art, in #3557 we used a wildcard and added a numeric prefix.  If you want to add one, then adding another patch in the future (if necessary) wouldn't require changing this line.

Reply:
Thanks for mentioning this. Funnily enough, I did attempt that approach and had to settle on this implementation. The wildcard and multiple patch approach ended up failing whichever secondary patch was being applied?

Reply:
Alright! Just thought I'd mention it as a possibility.